mobile-widgets/qml/DiveList.qml

```diff
@@ -313,7 +313,7 @@ Kirigami.ScrollablePage {
 		model: diveModel
 		currentIndex: -1
 		delegate: diveDelegate
-		//boundsBehavior: Flickable.StopAtBounds
+		boundsBehavior: Flickable.DragOverBounds
```

Comment:
That's really odd. I'll pull it, but I'll still disable Reachability.
New binaries coming up very soon.